dashboard/app/models/standard.rb

```diff
@@ -38,19 +40,30 @@ def summarize
 
   # @param filename [String] The path to the CSV file.
   # Expected columns:
-  # - organization
-  # - organization_id
+  # - framework_id
+  # - shortcode
   # - description
-  # - concept
+  # - organization (deprecated)
+  # - organization_id (deprecated)
+  # - concept (deprecated)
   def self.seed_from_csv(filename)
     created = 0
     updated = 0
+    framework = Framework.find_by!(shortcode: 'csta')
```

Comment:
Why csta?

Reply:
Ah I assume because that's the standard we currently have? Do you mind adding a comment here explaining that?

Reply:
@bethanyaconnor I have added a comment -- please let me know if it needs improvement.

Reply:
Yep looks good -- thanks!